website/docs/r/oracle_database_system.html.markdown

```diff
@@ -0,0 +1,174 @@
+  location = "eastus"
+}
+
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-virtual-network"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "example-subnet"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.0/24"]
+
+  delegation {
+    name = "delegation"
+
+    service_delegation {
+      actions = [
+        "Microsoft.Network/networkinterfaces/*",
+        "Microsoft.Network/virtualNetworks/subnets/join/action",
+      ]
+      name = "Oracle.Database/networkAttachments"
+    }
+  }
+}
+
+resource "azurerm_oracle_resource_anchor" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_oracle_network_anchor" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = "eastus"
+  resource_anchor_id  = azurerm_oracle_resource_anchor.example.id
+  subnet_id           = azurerm_subnet.example.id
+  zones               = ["2"]
+}
+
+resource "azurerm_oracle_database_system" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = "eastus"
+  zones               = ["1"]
+  cpu_core_count      = 4
+  source              = "None"
+  database_edition    = "StandardEdition"
+  database_version    = "19.27.0.0"
+  hostname            = "hostname"
+  network_anchor_id   = azurerm_oracle_network_anchor.example.id
+  resource_anchor_id  = azurerm_oracle_resource_anchor.example.id
+  shape               = "VM.Standard.x86"
+  ssh_public_keys     = ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+wWK73dCr+jgQOAxNsHAnNNNMEMWOHYEccp6wJm2gotpr9katuF/ZAdou5AaW1C61slRkHRkpRRX9FA9CYBiitZgvCCz+3nWNN7l/Up54Zps/pHWGZLHNJZRYyAB6j5yVLMVHIHriY49d/GZTZVNB8GoJv9Gakwc/fuEZYYl4YDFiGMBP///TzlI4jhiJzjKnEvqPFki5p2ZRJqcbCiF4pJrxUQR/RXqVFQdbRLZgYfJ8xGB878RENq3yQ39d8dVOkq4edbkzwcUmwwwkYVPIoDGsYLaRHnG+To7FvMeyO7xDVQkMKzopTQV8AuKpyvpqu0a9pWOMaiCyDytO7GGN you@me.com"]
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Database system. Changing this forces a new Database system to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Database system should exist. Changing this forces a new Database system to be created.
+
+* `location` - (Required) The Azure Region where the Database System should exist. Changing this forces a new Database system to be created.
+
+* `admin_password` - (Required) A strong password for SYS, SYSTEM, and PDB Admin.  Changing this forces a new Database system to be created.
```

Comment:
since `admin_password` is `required`, why it does not exist in `## Example Usage`?